src/sage/geometry/hyperplane_arrangement/arrangement.py

```diff
@@ -2904,8 +2952,9 @@ def whitney_data(self):
 
         EXAMPLES::
 
+            sage:# needs sage.combinat
```

Comment:
```suggestion
            sage: # needs sage.combinat
```
Is this space needed? At least this makes it more readable.